docs/topics/ref_known-issues-7-1-0.adoc

```diff
@@ -0,0 +1,13 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-14
+
+:_mod-docs-content-type: REFERENCE
+
+[id="known-issues-7-1-0_{context}"]
+= Known issues
+
+This section provides highlighted known issues in {ProductFullName} version 7.1.0.
+
+.Enabling or disabling Preemption does not work if authentication is enabled
+
+In {ProductFullName} 7.1.0, *Preemption* cannot be enabled or disabled for tasks in the Task Manager page and the Task Manager drawer if you installed {ProductShortName} with authentication enabled. Currently, there is no workaround available. (link:https://issues.redhat.com/browse/MTA-3195[MTA-3195])
```

Comment:
Hi @mpershina 
1. I don't think that you need to mention that disabling preemption doesn't work, because only enabled preemption task can be disabled. same in the title
2. MTA is authenticated by default. the W/A is to disable authentication (probably not recommended for MTA customer)

Suggestion:
Enabling Preemption does not work if authentication is enabled

In {ProductFullName} 7.1.0, **Preemption** cannot be enabled for tasks in the Task Manager page and the Task Manager drawer if authentication is enabled. The workaround is to disable the authentication.  (link:https://issues.redhat.com/browse/MTA-3195[MTA-3195])